content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,440 @@
+
+   ```shell
+   $ansible --version
+   ```
+
+For other operating systems, please refer to the official Ansible documentation.
+
+**Installing the HPE GreenLake Data Services Python SDK**
+
+These Ansible playbooks use the Python libraries of the HPE GreenLake Python SDK. Install the SDK using this command:
+
+**...Below command will change and use the newly available repository...**
+
+```shell
+$pip install git@github.hpe.com:sijeesh-kattumunda/greenlake_data_services_python.git
+```
+
+Cloning the DSCC GitHub repo	
+To clone the repo, execute the following command on the machine where you installed Ansible:
+
+**...Below command will change and use the newly available repository...**
+
+```shell
+$git clone git@github.hpe.com:sijeesh-kattumunda/greenlake-data-services-ansible.git
+```
+
+This repo mainly consists of two folders:
+
+1. **Libraries**: These are Python libraries that will be used by the Ansible playbooks to perform CRUD operations on DSCC resources.
+2. **Examples**: This folder consists of sample Ansible playbooks that perform CRUD operations on DSCC resources. With these examples, one can start building use cases.
+
+**Setting up environment variables**
+
+Set the ANSIBLE_LIBRARY variable to the greenlake\-data\-services\-ansible\/library directory of the repo -
+
+`ANSIBLE_LIBRARY=/home/admin/greenlake-data-services-ansible/library`
+
+Set the ANSIBLE\_MODULE\_UTILS to the module\_utils directory under greenlake\-data\-services\-ansible\/library directory -
+
+`ANSIBLE_MODULE_UTILS=/home/admin/greenlake-data-services-ansible/library/module_utils`
+
+**Usage**
+
+Let’s have a look at an example used to perform Data Services Cloud Console operations on a host using an Ansible playbook. For any Ansible playbook to be used, an inventory file is required. The Ansible inventory file defines the hosts and groups of hosts on which commands, modules, and tasks in a playbook operate. In this example, we are calling REST APIs from our local machine. This file can be placed anywhere and the path of this file can be given during the Ansible playbook execution. Create an inventory file, name it “hosts” (the name of the file can be anything), and update it with the following details: 
+
+
+`[localhost]
+127.0.0.1 ansible_connection=local ansible_python_interpreter=/home/admin/ansible-env/bin/python`
+
+The IP address given is localhost, which means operations are performed on the local host. Provide the Python interpreter location on your system. Name this file “hosts”.
+
+Let's look at a sample of the host file. Ansible playbooks contain a list\/array of tasks that will be performed sequentially. The below code snippet has the following tasks.
+
+1. **Creation** of host
+
+   In this block, the input request parameters are provided under the section ‘data’. In this sample, only required fields for the REST API call are provided such as name, initiator_ids, user_created flag, and operating system.
```

Comment:
In the preview, I don't think "usercreated flag" is showing properly. This is what I see:

![image](https://user-images.githubusercontent.com/77496261/228374758-1a5e37c3-180b-4204-bd04-d3f2890774b1.png)

Reply:
This is some issue with the escape characters. I had escaped them but they got reset. I'm not sure what went wrong there. Denis would help out on the underscore characters before publishing.